frontend/package-lock.json

```diff
@@ -1,12 +1,12 @@
 {
   "name": "zetaforge",
-  "version": "0.3.1",
+  "version": "0.3.2",
   "lockfileVersion": 3,
   "requires": true,
   "packages": {
     "": {
       "name": "zetaforge",
-      "version": "0.3.1",
```

Comment:
I think there's an issue with version because npm keeps wanting to upgrade my version as well.

Reply:
I think you should leave it like that though

Reply:
okay.